CRM/Report/Form/Case/Detail.php

```diff
@@ -180,6 +185,12 @@ public function __construct() {
             'operatorType' => CRM_Report_Form::OP_MULTISELECT,
             'options' => $this->rel_types,
           ),
+          'is_active' => array(
+            'title' => ts('Active Role?'),
+            'type' => CRM_Utils_Type::T_BOOLEAN,
+            'default' => TRUE,
+            'options' => $this->active_role_labels,
```

Comment:
Instead of adding this new class variable, how about just calling `CRM_Core_SelectValues::boolean()` here.